benchmarks/benchmarks.json

```diff
@@ -3,69 +3,59 @@
   "simple_copy": {
     "ref": {
       "type": "C++",
-      "source": "simple_copy/simple_copy.cc",
-      "iters": "1000",
-      "flags": ""
+      "benchmark": "bench_simple_copy",
+      "flags": []
     },
     "mlir": {
       "type": "MLIR",
-      "source": "../test/Benchmarks/simple_copy.mlir",
-      "iters": "1000",
-      "flags": ""
+      "benchmark": "simple_copy.mlir",
```

Comment:
Should these include file extension `.mlir`?

Reply:
I think so. This is to show (to developers adding benchmarks) that the benchmark in question is an MLIR file. Given that I have removed the path from the config, it's a bit more clear this way.